llvm/test/Transforms/LoopVectorize/reduction-reassociate.ll

```diff
@@ -0,0 +1,47 @@
+; Check that the loop with a floating-point reduction is vectorized
+; due to llvm.loop.vectorize.reassociate_fpreductions.enable metadata.
+; RUN: opt -passes=loop-vectorize -S < %s 2>&1 | FileCheck %s
+
+source_filename = "FIRModule"
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
If the test requires a X86 triple, it needs to be moved to the `X86` subdirectory.

You can drop `source_filename` and `target datalayout`.